app/backend/scripts/packageCC.sh

```
@@ -0,0 +1,44 @@
+CC_NAME=${1}
+CC_SRC_PATH=${2}
+CC_SRC_LANGUAGE=${3}
+CC_VERSION=${4}
+
+println "executing with the following"
+println "- CC_NAME: ${C_GREEN}${CC_NAME}${C_RESET}"
+println "- CC_SRC_PATH: ${C_GREEN}${CC_SRC_PATH}${C_RESET}"
+println "- CC_SRC_LANGUAGE: ${C_GREEN}${CC_SRC_LANGUAGE}${C_RESET}"
+println "- CC_VERSION: ${C_GREEN}${CC_VERSION}${C_RESET}"
+
+FABRIC_CFG_PATH=$PWD/../../fabric-samples/test-network/config/
```

Shellcheck (0.11.0)

[warning] FABRIC_CFG_PATH appears unused. Verify use (or export if used externally).

(SC2034)